File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program88.java
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept Character from user and check whether it is digit or not (0-9).
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Solution
{

	public char ch;

	public Solution(char ch)
	{
		this.ch = ch;
	}

	public boolean ChkDigit()
	{	
		
		if(ch>=48 && ch<=57)
		{
			return true;
		}
		else
		{
			return false;
		}
	}


}

class Program
{
	public static void main(String[] args)
	{
		Scanner sObj = new Scanner(System.in);
		boolean bRet = false;

		System.out.println("Enter character:");
		char ch = sObj.next().charAt(0);

		Solution obj = new Solution(ch);
		bRet = obj.ChkDigit();

		if(bRet == true)
		{
			System.out.println("It is Digit");
		}
		else
		{
			System.out.println("It is not Digit");
		}
	}
}

/*
OUTPUT:

Enter character:
7
It is Digit

*/

File: Logic_HandsOn/C++/Problems on Digits,String,Array,Pointers etc/program88.cpp
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept Character from user and check whether it is digit or not (0-9).
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<iostream>
using namespace std;

class Solution
{
	private:
		char ch;
		
	public:
		Solution(char ch='\0')
		{
			this->ch = ch;
		}

		bool ChkDigit()
		{
			if(ch>=48 and ch<=57)
			{
				return true;
			}
			else
			{
				return false;
			}


		}


};

int main()
{
	char ch='\0';
	bool bRet=false;

	cout<<"Enter the character: ";
	cin>>ch;

	Solution *dObj = new Solution(ch);
	bRet = dObj->ChkDigit();

	if(bRet==true)
	{
		cout<<"It is Digit\n";
	}
	else
	{
		cout<<"It is not Digit\n";
	}

	delete dObj;
	return 0;
}

/*
OUTPUT:

Enter the character: 7
It is Digit

*/
